magento2-nginx/fpm-7.0/usr/local/share/container/baseimage-30.sh

```diff
@@ -12,17 +14,25 @@ do_development_start() {
   do_magento2_development_start
 }
 
+alias_function do_templating do_magento2_templating_inner
+do_templating() {
+  mkdir -p /home/build/.hem/gems/
+  chown -R build:build /home/build/.hem/
```

Comment:
https://github.com/continuouspipe/dockerfiles/blob/master/ubuntu/16.04/Dockerfile#L35 sets this up but it didn't seem to help, unless this is a leftover from before that was added. I'll try it without when I get around to rebuilding.